### PR TITLE
New mod_ginger_embed

### PR DESCRIPTION
**Pull request by rl-king**
Replaces the Javascript generated embed code with an iframe. Sites are whitelisted in the site config with as comma separated list with an `allowed_hosts` key

### CODE REVIEW

**Comment by DorienD** [comment]
@ddeboer This is now ready to be merged.

File: modules/mod_ginger_embed/dispatch/dispatch [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [
-    {embed_js, ["embed", "embed.js"], controller_file, [{path, "js/embed.js"}, {root, [lib]}]},
+    {embed, ["gingerembed", id], controller_template, [{template, "embed.tpl"}]},
```
Comment (ddeboer) [inline modules/mod_ginger_embed/dispatch/dispatch]: Let’s not unnecessarily change the URL. The old one was fine (`["embed", id]`).

Comment (DorienD) [reply modules/mod_ginger_embed/dispatch/dispatch]: agreed

Comment (DorienD) [reply modules/mod_ginger_embed/dispatch/dispatch]: Done.

Comment (rl-king) [reply modules/mod_ginger_embed/dispatch/dispatch]: Agreed, there might have been some reasoning about it at the time, not sure now. If this is reverted this https://github.com/driebit/ginger/blob/95fa7da2f2bd299745f82402b1341f560659209f/modules/mod_ginger_embed/mod_ginger_embed.erl#L66 has to be modified as well.

Comment (DorienD) [reply modules/mod_ginger_embed/dispatch/dispatch]: shit! no we need ginger embed, because of the whitelisting

Comment (ddeboer) [reply modules/mod_ginger_embed/dispatch/dispatch]: No, just change the above snippet to`/embedd`, too.

Comment (DorienD) [reply modules/mod_ginger_embed/dispatch/dispatch]: ok

**Comment by ddeboer** [comment]
Missing the primary reason for the changes in the description: CSS scoped to the individual embed. Adding this to the commit message.

**Comment by ddeboer** [comment]
Thanks, all! We have a simpler embed now, unfortunately no longer used LD, but whatever.

**Pull request merged**